diff --git a/hello_world_rust_wasm_component_lib.js b/hello_world_rust_wasm_component_lib.js
index 824d8a1..c4a7706 100644
--- a/hello_world_rust_wasm_component_lib.js
+++ b/hello_world_rust_wasm_component_lib.js
@@ -124,7 +124,10 @@ function trampoline5(rep) {
   return handle;
 }
 
-const PString$1 = class PString{
+const PString$1 = class {
+  static {
+    Object.defineProperty(this, 'name', { value: 'PString', configurable: true });
+  }
   constructor(arg0) {
     if (!(arg0 instanceof PString)) {
       throw new Error('Resource error: Not a valid "HPString" resource.');
@@ -161,7 +164,10 @@ PString$1.prototype.call = function call(arg1) {
   }
 };
 
-const RString$1 = class RString{
+const RString$1 = class {
+  static {
+    Object.defineProperty(this, 'name', { value: 'RString', configurable: true });
+  }
   constructor(arg0) {
     if (!(arg0 instanceof RString)) {
       throw new Error('Resource error: Not a valid "HRString" resource.');
